### PR TITLE
Fix ddi formatting

### LINKED ISSUE #613: DDI Ratio plot "Observed vs Predicted/Observed"
Based on https://github.com/pchelle/Test-of-Qualification-DDI-UGT/blob/main/report.md

Horizontal lines at y=1, y=0.5 and y=2 are missing. And a line is wrongly inserted at y=0.3

Also it looks better if the plot is symmetrical in the Y direction, compare to

### PR DESCRIPTION
**Pull request by abdullahhamadeh**
- Increase size of tick font size
 - Fixes #613
 - Re: #614 Change horizontal/diagonal, 2-fold and Guest lines to black solid.

### CODE REVIEW

File: R/qualification-ddi.R [hunk 1]
```diff
@@ -217,11 +218,43 @@ generateDDIQualificationDDIPlot <- function(data) {
   ddiPlotConfiguration$export$height <- 2 * 1.2 * (data$plotSettings$height / 96)
   ddiPlotConfiguration$export$units <- "in"
 
+  # Set axis label font size
   ddiPlotConfiguration$labels$xlabel$font$size <- 2 * data$plotSettings$axisFontSize
   ddiPlotConfiguration$labels$ylabel$font$size <- 2 * data$plotSettings$axisFontSize
+
+  # Set axis tick font size
+  ddiPlotConfiguration$xAxis$font$size <- 2 * data$plotSettings$axisFontSize
```
Comment (msevestre) [inline R/qualification-ddi.R:226]: Do we agree that this is a quick fix until we have a centralized way for dealing with axis and font size?
@abdullahhamadeh and @pchelle ? 
I don't think that having some magic number spread out everywhere in the code (for each plot) will do us any good in the long run (and even in the short one)

File: R/qualification-ddi.R [hunk 2]
```diff
@@ -217,11 +218,43 @@ generateDDIQualificationDDIPlot <- function(data) {
   ddiPlotConfiguration$export$height <- 2 * 1.2 * (data$plotSettings$height / 96)
```
Comment (msevestre) [inline R/qualification-ddi.R:218]: or this for instance... We need a centralized way to deal with scaling. This is high prio IMO

File: R/qualification-ddi.R [hunk 3]
```diff
@@ -217,11 +218,43 @@ generateDDIQualificationDDIPlot <- function(data) {
...
+  if (data$axesSettings$X$scaling == "Log") {
+    ddiPlotConfiguration$xAxis$scale <- tlf::Scaling$log
+  }
+  if (data$axesSettings$Y$scaling == "Log") {
+    ddiPlotConfiguration$yAxis$scale <- tlf::Scaling$log
+  }
+
+  # Set y axis ticks and limits
+  if (residualsVsObservedFlag[[data$axesSettings$plotType]] & data$axesSettings$Y$scaling == "Log" ) {
+    log10Limit <- max(abs(sapply(c(floor, ceiling), function(fn) fn(log10(ddiData[[data$axesSettings$Y$label]])))))
```
Comment (msevestre) [inline R/qualification-ddi.R:249]: What does this line do?

File: R/qualification-ddi.R (hunk 1 above)
Comment (abdullahhamadeh) [reply R/qualification-ddi.R:226]: Yep.
Generally, multiplying configuration plan sizes by a factor of 2 seems to give outputs of appropriate proportions, comparable to what was produced in matlab.

File: R/qualification-ddi.R (hunk 3 above)
Comment (abdullahhamadeh) [reply R/qualification-ddi.R]: Calculation of the maximal y axis limit in log scale.  This has now been split into three lines:
1) Find lower extreme of log10(predict/observed), find its lower bounding y axis tick (in log10 scale)
2) Find upper extreme of log10(predict/observed), find its upper bounding y axis tick  (in log10 scale)
3) Maximal y axis limit is the maximum of the absolute values from 1 and 2.

**Comment by abdullahhamadeh** [comment]
[Report.docx](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/files/7584914/Report.docx)
Current status

**Comment by abdullahhamadeh** [comment]
@Yuri this is good to merge as discussed

**Comment by abdullahhamadeh** [comment]
@Yuri05 this is good to merge as discussed

**Pull request merged**